e2e/libs/backing_image/rest.py

```diff
@@ -9,12 +9,11 @@
 class Rest(Base):
 
     def __init__(self):
-        self.longhorn_client = get_longhorn_client()
         self.retry_count, self.retry_interval = get_retry_count_and_interval()
 
     def create(self, bi_name, source_type, url, expected_checksum):
         logging(f"Creating backing image {bi_name}")
-        self.longhorn_client.create_backing_image(
+        get_longhorn_client().create_backing_image(
```

Comment:
Since we moved the test back into k8s, the longhorn client is gotten from longhorn manager. after node reboot, longhorn manager is recreated and assigned a new IP, the old longhorn client has been invalid and needs to get it again.

Reply:
For code factor, I think some syntax is intended but we can refine it in the future.